docs/output.md

```diff
@@ -910,7 +910,9 @@ In the variant calling branch of the pipeline we are using [iVar trim](#ivar-tri
 
 - `assembly/<ASSEMBLER>/blastn/`
   - `*.blastn.txt`: BLAST results against the target virus.
-  - `*.filter.blastn.txt`: Filtered BLAST results.
+  - `*.filter.blastn.txt`: Filtered BLAST results. Applied filters by default are:
```

Comment:
You have removed this suffix.

Reply:
No, it is inside the `modules/local/filter_blastn.nf` process

Reply:
Yeah sorry missed that.